modular_lumos/code/game/machinery/fish_tanks.dm

```diff
@@ -0,0 +1,227 @@
+/obj/structure/fish_tank_base
+	name = "unfinished fish tank"
+	desc = "A tank that requires some glass"
+	icon = 'modular_lumos/icons/obj/fish_items.dmi'
+	icon_state = "tanki1"
+
+	density = TRUE
+
+/obj/structure/fish_tank_base/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/stack/sheet/glass))
+		var/obj/item/stack/sheet/glass/glass = I
+		if(glass.amount < 5)
+			return
+		if(!do_after(user, 2 SECONDS, FALSE, src))
+			return
+		glass.use(5)
+		new /obj/structure/fish_tank(get_turf(src))
+		qdel(src)
+		return
+	else if(istype(I, /obj/item/screwdriver))
+		I.play_tool_sound(src, 50)
+		if(!do_after(user, 2 SECONDS, FALSE, src))
+			return
+		I.play_tool_sound(src, 50)
+		var/obj/item/stack/sheet/metal/metal = new /obj/item/stack/sheet/metal(get_turf(src))
+		metal.amount = 20
```

Comment:
```suggestion
		metal.amount = 10
```
Construction needs 10 metal. You currently end up with 10 duplicated metal on disassembly.

Reply:
fixed